powerlevel9k.zsh-theme

```diff
@@ -744,9 +744,11 @@ prompt_docker_machine() {
 # GO prompt
 prompt_go_version() {
   local go_version
+  local go_path
   go_version=$(go version 2>/dev/null | sed -E "s/.*(go[0-9.]*).*/\1/")
+  go_path=$(go env GOPATH)
 
-  if [[ -n "$go_version" ]]; then
+  if [[ -n "$go_version" && "${PWD##$go_path}" != "$PWD" ]]; then
```

Comment:
Hm, so this is interesting. `GOPATH` being empty doesn't necessarily mean that golang isn't installed, but as far as P9k is concerned, it might as well not be. I think the best answer is that if `GOPATH` is empty the segment is not printed.

@zikes - Do you need help modifying the test per @dritter's comment? I do want to get this PR merged as soon as we can =)